locksmith/src/fulfillment/tokenTransferApproval.ts

```diff
@@ -0,0 +1,26 @@
+import { ethers } from 'ethers'
+
+export default class TokenTransferApproval {
+  provider: ethers.providers.JsonRpcProvider
+  privateKey: string
+
+  constructor(provider: string, credentials: string) {
```

Comment:
```suggestion
  constructor(provider: string, privateKey: string) {
```

Not a huge deal, but might make it more clear at call sites what goes here when using code completion

Reply:
sounds good, will make the update